.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea/
```

Comment:
Please remove this, you can use the global .gitignore instead 👍

Reply:
Done, thanks for the link.